src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -95,7 +105,7 @@ def __init__(self, config):
             self.rotary_ndims, config.max_position_embeddings, base=config.rotary_emb_base
         )
         self.norm_factor = torch.sqrt(torch.tensor(self.head_size, dtype=torch.float32)).to(torch.get_default_dtype())
-        self.query_key_value = nn.Linear(config.hidden_size, 3 * config.hidden_size)
+        self.query_key_value = LoRALinear(config.hidden_size, 3 * config.hidden_size, "selfattn", config)
```

Comment:
As GPT-NeoX groups all attention matrices together into one linear layers similar to GPT-2, this should probably use `LoRAMergedLinear` instead of `LoRALinear` if i'm not mistaken (see GPT-2 implementation for reference).

Reply:
I fixed this already!